src/select-field.jsx

```diff
@@ -36,6 +36,7 @@ const SelectField = React.createClass({
     floatingLabelText: React.PropTypes.node,
     fullWidth: React.PropTypes.bool,
     hintText: React.PropTypes.node,
+    hintStyle: React.PropTypes.object,
```

Comment:
@kodermax Prop types need to be alphabetical. Just put it above hintText and the error should go away